lib/begin.go

```diff
@@ -188,6 +189,22 @@ func (a *ACBuild) writeEmptyManifest() error {
 		return err
 	}
 
+	archvalue := runtime.GOARCH
+	if runtime.GOOS == "linux" && archvalue == "arm" {
+		archvalue = "armv7l"
+		walker := func(path string, info os.FileInfo, err error) error {
```

Comment:
We used the same trick in mesos-dns: https://github.com/mesosphere/mesos-dns/blob/883e8ace58008d18f35bca878175212eea5b6b1f/detect/masters.go#L107

Maybe we could factor this in a library function returning `binary.ByteOrder`?